app/models/post.rb

```diff
@@ -10,7 +10,6 @@ def recent_comments
   after_save :update_posts_counter
```

Comment:
- [ ] Also in line number 4 the association should be attributed to the User class name.

Reply:
Done :white_check_mark: